src/Tickets/Emails/Settings.php

```diff
@@ -0,0 +1,283 @@
+
+	/**
+	 * Gets the template instance used to setup the rendering html.
+	 *
+	 * @since TBD
+	 *
+	 * @return Tribe__Template
+	 */
+	public function get_template() {
+		if ( empty( $this->template ) ) {
+			$this->template = new Tribe__Template();
+			$this->template->set_template_origin( Tribe__Tickets__Main::instance() );
+			$this->template->set_template_folder( 'src/admin-views/settings/tickets-emails' );
+			$this->template->set_template_context_extract( true );
+		}
+
+		return $this->template;
+	}
+
+	/**
+	 * Adds list of Templates to the Tickets Emails settings tab.
+	 * 
+	 * @param  [] $fields Current array of Tickets Emails settings fields.
+	 * 
+	 * @return [] $fields Filtered array of Tickets Emails settings fields.
+	 */
+	public function add_template_list( $fields ) {
+
+		$template = $this->get_template();
+
+		// @todo Replace this with array of actual Message Template objects that do not yet exist.
+		$templates = [
+			[
+				'title'     => 'Ticket Email',
+				'enabled'   => true,
+				'recipient' => 'Purchaser',
+			],
+			[
+				'title'     => 'RSVP Email',
+				'enabled'   => true,
+				'recipient' => 'Attendee',
+			],
+			[
+				'title'     => 'Order Notification',
+				'enabled'   => false,
+				'recipient' => 'Site Admin',
+			],
+			[
+				'title'     => 'Order Failure',
+				'enabled'   => true,
+				'recipient' => 'Site Admin',
+			],
+		];
+
+		$new_fields = [
+			[
+				'type' => 'html',
+				'html' => $template->template( 'message-templates', [ 'templates' => $templates ], false ),
+			],
+		];
+
+		$new_fields = apply_filters( 'tec_tickets_emails_settings_template_list', $new_fields );
+
+		return array_merge( $fields, $new_fields );
+	}
+
+	/**
+	 * Adds Sender Info fields to Tickets Emails settings.
+	 *
+	 * @param  [] $fields Current array of Tickets Emails settings fields.
+	 * 
+	 * @return [] $fields Filtered array of Tickets Emails settings fields.
+	 */
+	public function sender_info_fields( $fields ) {
+
+		$current_user = get_user_by( 'id', get_current_user_id() );
+
+		$new_fields = [
+			[
+				'type' => 'html',
+				'html' => '<h3>' . esc_html__( 'Sender Information', 'event-tickets' ) . '</h3>',
+			],
+			[
+				'type' => 'html',
+				'html' => '<p>' . esc_html__( 'If fields are empty, sender information will be from the site owner set in WordPress general settings.', 'event-tickets' ) . '</p>',
+			],
+			static::$option_sender_name  => [
+				'type'                => 'text',
+				'label'               => esc_html__( 'Sender Name', 'event-tickets' ),
+				'size'                => 'medium',
+				'default'             => $current_user->user_nicename,
+				'validation_callback' => 'is_string',
+				'validation_type'     => 'textarea',
+			],
+			static::$option_sender_email  => [
+				'type'                => 'text',
+				'label'               => esc_html__( 'Sender Email', 'event-tickets' ),
+				'size'                => 'medium',
+				'default'             => $current_user->user_email,
```

Comment:
I think we have this one living in `tickets-commerce-confirmation-email-sender-email`. We'll need to find a migration path so that if a user already set that up, it's being populated with the saved details.